.github/workflows/pre-release.yml

```diff
@@ -5,11 +5,11 @@ on:
   workflow_dispatch:
```

Comment:
There are a lot of unrelated formatting changes in this commit.

If you want them to be part of the PR can you at least split up the commit between formatting and the log change so the changes make sense in the future?

Reply:
your comment makes sense, but it's just formatting from prettier and it corrects spaces and indentation correctly, but okay

Reply:
https://github.com/metabase/metabase/pull/41497